components/fieldlabel/stories/form.stories.js

```diff
@@ -65,7 +85,26 @@ Default.args = {
 				}, context),
 			],
 		}, {
-			label: "Amount",
+			label: "Interest",
+			id: "form-example-interests",
+			content: [
+				(passthroughs, context) => Fieldgroup({
+					layout: "horizontal",
+					items: [
+						Checkbox({
+							...passthroughs,
+							label: "Kittens",
+							customClasses: ["spectrum-FieldGroup-item"],
+						}, context),
+						Checkbox({
+							...passthroughs,
+							label: "Puppies",
+							customClasses: ["spectrum-FieldGroup-item"],
+						}, context),]
+				}),
+			],
```

Comment:
I don't have default `items` if a user tries to use a form. Should I create some, sort of like [what was mentioned for menu](https://github.com/adobe/spectrum-css/pull/3134#discussion_r1773642778)?

Reply:
I don't have a strong opinion on this either way, but it's probably a little smoother if you move these items up to the default args rather than being the args for the Default story, especially since you're reusing them for all the other stories.